notary-server/README.md

```diff
@@ -117,6 +117,10 @@ Currently, both the private key (and cert) used to establish TLS connection with
 #### Authorization
 An optional authorization module is available to only allow requests with valid API key attached in the authorization header. The API key whitelist path (as well as the flag to enable/disable this module) can be changed in the config (`authorization` field).
 
+Hot reloading of the whitelist is supported, i.e. modification of the whitelist file will be automatically applied without needing to restart the server. Please take note of the following
+- Avoid using auto save mode when editing the whitelist to prevent spamming hot reloads
```

Comment:
This is perfectly fine. Production servers will push changes.

Reply:
@heeckhau what do you mean by 'push changes' ya?

Reply:
AFAIK devops do not edit config files on live servers. Instead they commit changes, review them and push them to the server

Reply:
ah i see, i was imaging these whitelist files to not be tracked by git, since they contain secrets like api keys, so they would be edited somewhere secretly whether on or off servers.